drivers/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_subdirectory_ifdef(CONFIG_SENSOR sensor)
 add_subdirectory_ifdef(CONFIG_NETWORKING net)
 add_subdirectory_ifdef(CONFIG_BT_LL_SOFTDEVICE bt_ll_softdevice)
 add_subdirectory_ifdef(CONFIG_MPSL mpsl)
+add_subdirectory_ifdef(CONFIG_NRF_LPUART serial)
```

Comment:
It should be `add_subdirectory(serial)` here and `zephyr_library_sources_ifdef(...` in `serial/`.